etw/rust/tracelogging/src/native.rs

```diff
@@ -69,10 +69,14 @@ impl ProviderContext {
         {
             result = 50; // ERROR_NOT_SUPPORTED
         }
-        #[cfg(all(windows, feature = "etw"))]
+        #[cfg(all(windows, feature = "etw", feature = "user_mode"))]
         {
             result = unsafe { EventActivityIdControl(_control_code, _activity_id) };
         }
+        #[cfg(all(windows, feature = "etw", feature = "kernel_mode"))]
+        {
+            result = unsafe { EtwActivityIdControl(_control_code, _activity_id) };
+        }
```

Comment:
Please add a new value WindowsKernelMode to the NativeImplementation enumeration, and set the `NATIVE_IMPLEMENTATION` constant accordingly.

Reply:
Added WindowsKernelMode enum and set it in NATIVE_IMPLEMENTATION const